docs/vscode-setup.md

```diff
@@ -23,4 +25,4 @@ If you want to use Visual Studio Code to develop and debug then follow these ste
         "projectName": "gateway-service"
     }
     ```
-Launch either of the services in debug mode and then attach the debugger by running the appropriate launch configuration.
\ No newline at end of file
+4. Launch either of the services in debug mode. Then attach the debugger by running the appropriate launch configuration.
```

Comment:
Should we specify what the appropriate launch configurations could be?

Reply:
This document has an request to community to enhance https://github.com/zowe/api-layer/issues/666